### ISSUE TITLE
Cannot properly parse arguments with spaces

### ISSUE THREAD

**Issue by yuk7** [yuk7/wsl2exe#1]
Arguments containing spaces will be split.
I exec:
```cmd
wsl2exe.exe "C:\Program Files\Microsoft VS Code\bin\code"
```
exe runs:
```bash
wsl2exe "C:\Program" "Files\Microsoft" "VS" "Code\bin\code"
```

**Comment by yuk7** [comment]
Fixed in ff377b0

**Comment by yuk7** [comment]
Fixed in 039060f
